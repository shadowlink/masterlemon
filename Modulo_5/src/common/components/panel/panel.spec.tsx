import * as React from 'react';
import { shallow } from 'enzyme';
import { Panel } from './panel';

describe('commom/components/panel tests', () => {
  it('should render as expected when passing required properties', () => {
    // Arrange
    const props = {
      title: 'test title',
    };

    // Act
    const component = shallow(
      <Panel {...props}>
        <h1>Test children component</h1>
      </Panel>
    );

    // Assert
    expect(component).toMatchSnapshot();
  });
});
