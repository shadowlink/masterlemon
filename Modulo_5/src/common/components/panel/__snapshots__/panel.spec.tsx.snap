// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commom/components/panel tests should render as expected when passing required properties 1`] = `
<div
  className="card"
>
  <Component
    title="test title"
  />
  <Component>
    <h1>
      Test children component
    </h1>
  </Component>
</div>
`;
